components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -68,6 +76,48 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
         this.props.actions.getLicenseConfig();
     }
 
+    renderStartTrial = (learnMoreURL: string, gettingTrialError: React.ReactNode) => {
+        return (
+            <React.Fragment>
```

Comment:
Copied from Hossein's comment: we can use `<> </>` as a short hand for react fragment.

Reply:
I think it's really subjective unless we have a project-wide convention. Either variant has no advantage over the other and like I mentioned in the other review comment, `<>` breaks tag matching in my IDE so that's 0:1 for `<>` vs `React.Fragment`.